cnf-tests/testsuites/e2esuite/ptp/ptp.go

```
@@ -31,10 +32,26 @@ const (
 	ptpDaemonsetName         = "linuxptp-daemon"
 )
 
+var (
+	isSingleNode bool
+)
+
+func init() {
+	singleNodeCluster := os.Getenv("SINGLE_NODE_CLUSTER")
+	if singleNodeCluster != "" {
+		isSingleNode, _ = strconv.ParseBool(singleNodeCluster)
```

Can you Expect no error here instead of swallowing it?